x/protocolpool/keeper/keeper.go

```diff
@@ -177,6 +177,31 @@ func (k Keeper) SetToDistribute(ctx context.Context) error {
 	// Calculate the amount to be distributed
 	amountToDistribute := distributionBalance.Sub(lastBalance)
 
+	// Check if there are any recipients to distribute to, if not, send straight to the community pool and avoid
+	// setting the distributions
+	hasContinuousFunds := false
+	err = k.ContinuousFund.Walk(ctx, nil, func(_ sdk.AccAddress, _ types.ContinuousFund) (bool, error) {
+		hasContinuousFunds = true
+		return true, nil
```

Comment:
we are going to stop the iteration anyway, then should there be an easier way to set `hasContinuousFunds` without do it inside a Walk function ?

Reply:
Not sure what you mean. I think there's no other way to check if there is anything in a collections' Map